src/js/injectBaseStylesheet.js

```diff
@@ -55,13 +55,14 @@ export default function injectBaseStylesheet() {
     return;
   }
 
+  // create the injected styles
   const styleEl = document.createElement("style");
-  styleEl.type = "text/css";
   styleEl.classList = "drift-base-styles";
-
   styleEl.textContent = RULES;
-  const head = document.head;
 
+  // prepend them to the document head's first child
+  const head = document.head;
   const allHeadElements = head.getElementsByTagName("*");
-  allHeadElements.innerHTML = styleEl + allHeadElements.innerHTML;
+  const firstItem = allHeadElements.item(0);
+  firstItem.outerHTML = styleEl.outerHTML + firstItem.outerHTML;
```

Comment:
If we can't get around the CSP violation anyway, is it cleaner to go back to using `.insertBefore()` rather than this method? i.e. reverting the changes introduced [here](https://github.com/imgix/drift/pull/598/files)?

Reply:
That's a good point. I do think this read easier that way.. 

It might then be better to close this PR altogether and create two new ones: 

1) rolling back that change and then
2) adding the CSP to `index.html` as an example / check for handling those issues.

I'll discuss w/ @sherwinski offline and circle back.